### PR TITLE
RHVH FISMA Low profile

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
@redhatrises once build system is updated to build rhv4 content will be able to test for missing variables etc. What do you think of current state?

File: rhv4/profiles/rhvh-vpp.profile [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    - require_singleuser_auth
+    - accounts_no_uid_except_zero
+    - no_direct_root_logins
+    - no_password_auth_for_systemaccounts
+    - restrict_serial_port_logins
+    - securetty_root_login_console_only
+    - sshd_disable_root_login
+
+    # IA-2 (1)
+    - smartcard_auth
```
Comment (redhatrises) [inline rhv4/profiles/rhvh-vpp.profile]: @shawndwells please remove this rule and replace with the `opensc` rules

File: rhel7/profiles/rhelh-vpp.profile [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    - require_singleuser_auth
+    - accounts_no_uid_except_zero
+    - no_direct_root_logins
+    - no_password_auth_for_systemaccounts
+    - restrict_serial_port_logins
+    - securetty_root_login_console_only
+    - sshd_disable_root_login
+
+    # IA-2 (1)
+    - smartcard_auth
```
Comment (redhatrises) [inline rhel7/profiles/rhelh-vpp.profile]: @shawndwells please remove this rule and replace with the `opensc` rules

**Comment by redhatrises** [comment]
Ack on @shawndwells changes.

File: rhel7/profiles/rhelh-vpp.profile [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    - U.S. Government Configuration Baseline (USGCB)
+    - NIAP Protection Profile for Virtualization v1.0 (VPP v1.0)
+
+    For any differing configuration requirements, e.g. password lengths, the stricter
+    security setting was chosen. Security Requirement Traceability Guides (RTMs) and
+    sample System Security Configuration Guides are provided via the
+    scap-security-guide-docs package.
+
+    This profile reflects U.S. Government consensus content and is developed through
+    the OpenSCAP/SCAP Security Guide initiative, championed by the National
```
Comment (shawndwells) [inline rhel7/profiles/rhelh-vpp.profile]:
```suggestion
    the ComplianceAsCode project, championed by the National
```

File: rhel7/profiles/rhelh-vpp.profile [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    Security Agency. Except for differences in formatting to accommodate
+    publishing processes, this profile mirrors OpenSCAP/SCAP Security Guide
```
Comment (shawndwells) [inline rhel7/profiles/rhelh-vpp.profile]:
```suggestion
    publishing processes, this profile mirrors ComplianceAsCode
```

File: rhv4/profiles/rhvh-vpp.profile [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+    For any differing configuration requirements, e.g. password lengths, the stricter
+    security setting was chosen. Security Requirement Traceability Guides (RTMs) and
+    sample System Security Configuration Guides are provided via the
+    scap-security-guide-docs package.
+
+    This profile reflects U.S. Government consensus content and is developed through
+    the OpenSCAP/SCAP Security Guide initiative, championed by the National
+    Security Agency. Except for differences in formatting to accommodate
+    publishing processes, this profile mirrors OpenSCAP/SCAP Security Guide
```
Comment (shawndwells) [inline rhv4/profiles/rhvh-vpp.profile]:
```suggestion
    publishing processes, this profile mirrors ComplianceAsCode
```

File: rhv4/profiles/rhvh-vpp.profile [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+    - U.S. Government Configuration Baseline (USGCB)
+    - NIAP Protection Profile for Virtualization v1.0 (VPP v1.0)
+
+    For any differing configuration requirements, e.g. password lengths, the stricter
+    security setting was chosen. Security Requirement Traceability Guides (RTMs) and
+    sample System Security Configuration Guides are provided via the
+    scap-security-guide-docs package.
+
+    This profile reflects U.S. Government consensus content and is developed through
+    the OpenSCAP/SCAP Security Guide initiative, championed by the National
```
Comment (shawndwells) [inline rhv4/profiles/rhvh-vpp.profile]:
```suggestion
    the ComplianceAsCode project, championed by the National
```

File: rhel7/profiles/rhelh-vpp.profile [hunk 7]
```diff
@@ -0,0 +1,218 @@
...
+    of Standards and Technology (NIST), U.S. Department of Defense,
+    the National Security Agency, and Red Hat.
+
+    This baseline implements configuration requirements from the following
+    sources:
+
+    - Committee on National Security Systems Instruction No. 1253 (CNSSI 1253)
+    - NIST 800-53 control selections for MODERATE impact systems (NIST 800-53)
+    - U.S. Government Configuration Baseline (USGCB)
+    - NIAP Protection Profile for Virtualization v1.0 (VPP v1.0)
```
Comment (shawndwells) [inline rhel7/profiles/rhelh-vpp.profile]:
```suggestion
    - U.S. Government/NIAP Protection Profile for Virtualization v1.0 (PPBV v1.0)
```

Comment (shawndwells) [reply rhel7/profiles/rhelh-vpp.profile]: Not sure to call this VPP or PPBV.
NIAP has short name as "pp_base_virtualization" but other places it's "VPP"

File: rhv4/profiles/rhvh-vpp.profile [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+    of Standards and Technology (NIST), U.S. Department of Defense,
+    the National Security Agency, and Red Hat.
+
+    This baseline implements configuration requirements from the following
+    sources:
+
+    - Committee on National Security Systems Instruction No. 1253 (CNSSI 1253)
+    - NIST 800-53 control selections for MODERATE impact systems (NIST 800-53)
+    - U.S. Government Configuration Baseline (USGCB)
+    - NIAP Protection Profile for Virtualization v1.0 (VPP v1.0)
```
Comment (shawndwells) [inline rhv4/profiles/rhvh-vpp.profile]:
```suggestion
    - U.S. Government/NIAP Protection Profile for Virtualization v1.0 (PPBV v1.0)
```

File: rhel7/profiles/rhelh-vpp.profile (hunk 7 above)
Comment (redhatrises) [reply rhel7/profiles/rhelh-vpp.profile]: Let's keep is as VPP... That's what is on the title page.

File: rhv4/profiles/rhvh-vpp.profile (hunk 8 above)
Comment (redhatrises) [reply rhv4/profiles/rhvh-vpp.profile]: Let's keep is as VPP... That's what is on the title page.

**Comment by shawndwells** [comment]
LGTM and the build passes

**Comment by redhatrises** [comment]
Merging since Shawn reviewed my changes, and I reviewed his changes.

**Pull request merged**